cat-theme/src/css/header/_navigation.scss

```diff
@@ -0,0 +1,225 @@
+$background--color:#e6e6e6;
+$icon--color:#1e2023;
+$font--color:#ffffff;
+$font--color--active:#000000;
+$icon--size: 3rem;
+$font--primary:'Fira Sans',
+sans-serif;
+$transition--length: .8;
```

Comment:
Why are there variables defined here when there is also a `variables.scss` file where most of the variables are defined?  We should consolidate the variables in 1 location.